generator/src/test/java/com/scottlogic/deg/generator/validators/GenerationConfigValidatorTests.java

```diff
@@ -124,10 +134,56 @@ public void fullSequentialWithMaxRowsReturnsValid() {
         GenerationConfigValidator validator = new GenerationConfigValidator();
 
         //Act
-        ValidationResult validationResult = validator.validateCommandLine(config);
+        ValidationResult validationResult = validator.validateCommandLine(config, fileOutputTarget);
 
         //Assert
         Assert.assertTrue(validationResult.isValid());
         Assert.assertThat(validationResult.errorMessages, is(empty()));
     }
+
+    @Test
+    public void outputFileAlreadyExists() throws IOException {
+        //Arrange
+        TestGenerationConfigSource testConfigSource = new TestGenerationConfigSource(
+            GenerationConfig.DataGenerationType.FULL_SEQUENTIAL,
+            GenerationConfig.TreeWalkerType.REDUCTIVE,
+            GenerationConfig.CombinationStrategyType.EXHAUSTIVE
+        );
+        testConfigSource.setMaxRows(1234567L);
+        GenerationConfig config = new GenerationConfig(testConfigSource);
+        GenerationConfigValidator validator = new GenerationConfigValidator();
+
+        Path tmpPath = Files.createTempFile("datahelix-test", "csv");
```

Comment:
See previous comment about injecting a `FileSystem`. This might help in tests where you wouldn't need to create a file to assert behaviour?